blue_modules/libportal-react-native/example/metro.config.js

```
@@ -0,0 +1,45 @@
+const { getDefaultConfig, mergeConfig } = require('@react-native/metro-config');
+const path = require('path');
+const escape = require('escape-string-regexp');
```

Biome

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)